CDS/scripts/aspera.sh

```diff
@@ -65,29 +69,64 @@ fi
 
 export ASPERA_SCP_PASS=$PASSWORD
 
+RESULT=0
+
 if [ "$cf_media_file" != "" ]; then
 	ascp --host=$HOST --user=$USERNAME --mode=send $EXTRAFLAGS $cf_media_file $REMOTEPATH/$UPLOAD_MEDIA
+	
+	CODE=$?
+
+	if [ "$CODE" != "0" ]; then
+       echo -ERROR: Aspera encountered an error $? uploading the media file.
+       $RESULT=1
+    else
+		echo +SUCCESS: Aspera media file upload completed successfully.
+	fi
 fi
 
 if [ "$cf_meta_file" != "" ]; then
 	ascp --host=$HOST --user=$USERNAME --mode=send $EXTRAFLAGS $cf_meta_file $REMOTEPATH/$UPLOAD_META
+	
+	CODE=$?
+
+	if [ "$CODE" != "0" ]; then
+       echo -ERROR: Aspera encountered an error $? uploading the meta file.
+       $RESULT=1
+    else
+		echo +SUCCESS: Aspera meta file upload completed successfully.
+	fi
 fi
 
 if [ "$cf_inmeta_file" != "" ]; then
 	ascp --host=$HOST --user=$USERNAME --mode=send $EXTRAFLAGS $cf_inmeta_file $REMOTEPATH/$UPLOAD_INMETA
+	
+	CODE=$?
+
+	if [ "$CODE" != "0" ]; then
+       echo -ERROR: Aspera encountered an error $? uploading the inmeta file.
+       $RESULT=1
+    else
+		echo +SUCCESS: Aspera inmeta file upload completed successfully.
+	fi
 fi
 
 if [ "$cf_xml_file" != "" ]; then
 	ascp --host=$HOST --user=$USERNAME --mode=send $EXTRAFLAGS $cf_xml_file $REMOTEPATH/$UPLOAD_XML
+	
+	CODE=$?
+
+	if [ "$CODE" != "0" ]; then
+       echo -ERROR: Aspera encountered an error $? uploading the XML file.
+       $RESULT=1
+    else
+		echo +SUCCESS: Aspera XML file upload completed successfully.
+	fi
```

Comment:
it's not very DRY - repeated copy/paste - and i'd prefer it as a function that's called but it'll do for now :)